examples/ansible/setup_libvirt_host.yml

```diff
@@ -125,5 +122,10 @@
     - name: Report running Libvirt VMs, if any
       ansible.builtin.debug:
         msg: "Running VMs: {{ virt_list_vms.list_vms | join(', ') }}"
-      when:
-        - ((virt_list_vms.list_vms | default([])) | length) > 0
+      when: ((virt_list_vms.list_vms | default([])) | length) > 0
```

Comment:
Unnecessary formatting/whitespace change.

Reply:
followed the same coding style/pattern as the original code